tests/rptest/clients/kcl.py

```diff
@@ -241,6 +245,11 @@ def unfold(x):
         if len(topic_partitions_offsets) == 0:
             return []
 
+        if validate_offsets(topic_partitions_offsets) is False:
+            raise RuntimeError(
+                f'Negative offset passed to delete-records: {topic_partitions_offsets}'
+            )
```

Comment:
Don't we accept -1? https://github.com/redpanda-data/redpanda/blob/845a728fbc8807348f2ac88fd2a9e03063a05c00/src/v/kafka/server/handlers/delete_records.cc#L132

Reply:
Replied to above